### ISSUE TITLE
Having issue with enabling Xbox live in dashboard and in unity.

### ISSUE THREAD

**Issue by sulemanmuneer4212** [microsoft/xbox-live-unity-plugin#300]
Hi
I am having an issue with Xbox live. When I try to "Run Xbox Live Association Wizard" it shows me this error.
![Annotation 2020-05-14 143557](https://user-images.githubusercontent.com/65340070/81919566-6f964f00-95f1-11ea-94fe-73958d993ad5.png)

And on partner center dashboard currently it is showing this error.
![Annotation 2020-05-14 144548](https://user-images.githubusercontent.com/65340070/81919967-f2b7a500-95f1-11ea-82f1-cd405b82a8a1.png)

Please help me what could be the possible reasons for these and what could be the possible solutions.

**Comment by cookiedot** [comment]
I've exactly the same... :(

**Comment by sulemanmuneer4212** [comment]
@cookiedot  Hi. Is your issue resolved? If yes kindly help me out.

**Comment by cookiedot** [comment]
Yeah, the warning is gone after 2 days.. Certainly a delay to enable xbox live functionnalities..
For the Xbox live Association Wizard, it seem's to be broken. I've made the conf by hand (you can open conf file on Unity -xbox live menu- and fill the form with your account data)
